src/test/java/de/adesso/kicker/notification/MatchVerificationRequest/VerifyMatchServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package de.adesso.kicker.notification.MatchVerificationRequest;
+
+import de.adesso.kicker.match.Match;
+import de.adesso.kicker.match.MatchDummy;
+import de.adesso.kicker.user.UserService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class VerifyMatchServiceTest {
+
+    @Mock
+    MatchVerificationRequestRepository matchVerificationRequestRepository;
+
+    @Mock
+    UserService userService;
+
+    @InjectMocks
+    VerifyMatchService verifyMatchService;
+
+    static MatchVerificationRequest createMatchVerification() {
+        return MatchVerificationRequestDummy.matchVerificationRequest();
+    }
+
+    static List<MatchVerificationRequest> createMatchVerificationList() {
+        return Collections.singletonList(MatchVerificationRequestDummy.matchVerificationRequest());
+    }
+
+    @BeforeAll
```

Comment:
Please use `@BeforeEach` and remove `@TestInstance(TestInstance.LifeCycle.PER_CLASS)` so tests are independent from the class itself and are tested on their own.

Reply:
Alright, did this because it was used like this in other tests

Reply:
Keep your Master-Branch up do date, yes we haved used it too but we've changed it some days ago.